test/integration/scripts/test_stats.py

```diff
@@ -20,10 +20,12 @@
 from pathlib import Path
 
 from autosubmit.scripts.autosubmit import main
+from test.integration.conftest import sleep
 
 _EXPID = 't000'
 
 
+@sleep(1)
```

Comment:
Ah, I thought about a different way to fix this test, that I think could be even more deterministic.

```
$ python3
Python 3.10.12 (main, Aug 15 2025, 14:32:43) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import multiprocessing
>>> multiprocessing.active_children()
[]
>>> 
```

Or maybe something with psutil. When you run the `autosubmit run` we need to ensure that it finished and the log recovery process, or any other child process (jobs? 🤷 ) have completed. So instead of simply sleeping, you can implement a timeout of 10 or 30 seconds that waits until all processes have finished or fail with a nice message saying "Waited 30 seconds but processes [151561, 323232, etc] are still alive!". That way we would know why the test failed.